packages/plugin-board/src/index.js

```diff
@@ -48,6 +47,21 @@ registerPlugin(`board`, Board, {
         extract(options) {
           return Promise.resolve(options.body);
         }
+      },
+      {
+        name: `encryptChannel`,
+        direction: `outbound`,
+
+        test(ctx, options) {
+          if (get(options, `service`) === `board` && has(options, `body.aclUrlLink`)) {
```

Comment:
In this case, channel will only have {aclUrlLink, kmsMessage} which is not a very good identifier. 
I'll change it to `ctx.spark.device.isSpecificService(`board`, options.uri)`. :)